packages/dashboard-v2/src/components/Uploader/buildUploadErrorMessage.js

```diff
@@ -0,0 +1,37 @@
+import { getReasonPhrase } from "http-status-codes";
+import bytes from "pretty-bytes";
+
+export default function buildUploadErrorMessage(error) {
+  // The request was made and the server responded with a status code that falls out of the range of 2xx
+  if (error.response) {
+    if (error.response.data.message) {
+      return `Upload failed with error: ${error.response.data.message}`;
+    }
+
+    const statusCode = error.response.status;
+    const statusText = getReasonPhrase(error.response.status);
```

Comment:
you can use `statusCode` here

Reply:
Thanks for this -- I'll address it in future PRs to not cause a chain of changes between this branch and other PRs that depend on it 👌